test/e2e/storage/persistent_volumes-local.go

```diff
@@ -237,9 +440,41 @@ func createLocalPVCPV(config *localTestConfig, volume *localTestVolume) {
 	var err error
 	volume.pv, volume.pvc, err = framework.CreatePVPVC(config.client, pvConfig, pvcConfig, config.ns, true)
 	framework.ExpectNoError(err)
-	framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc)
+	framework.ExpectNoError(framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc))
 }
 
-func createLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
+func makeLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
 	return framework.MakePod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd)
 }
+
+func createLocalPod(config *localTestConfig, volume *localTestVolume) (*v1.Pod, error) {
+	return framework.CreatePod(config.client, config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, "")
+}
+
+// Create corresponding write and read commands
+// to be executed inside containers with local PV attached
+func createWriteAndReadCmds(testFileDir string, testFile string, writeTestFileContent string) (writeCmd string, readCmd string) {
+	testFilePath := filepath.Join(testFileDir, testFile)
+	writeCmd = fmt.Sprintf("mkdir -p %s; echo %s > %s", testFileDir, writeTestFileContent, testFilePath)
+	readCmd = fmt.Sprintf("cat %s", testFilePath)
+	return writeCmd, readCmd
+}
+
+// Execute a read or write command in a pod
+func podRWCmdExec(pod *v1.Pod, cmd string) string {
+	out, err := podExec(pod, cmd)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Just an observation - in createLocalPod, the error is returned; here the error is handled in scope.  @jeffvance and I have found it's generally better to return errors and handle them at the It() scope.